### PR TITLE
Odim 65 (JSONBody in task response) task 4: svc- events changes

### PR DESCRIPTION
**Pull request by a-ajith**

### CODE REVIEW

File: svc-events/events/evtsubscription.go [hunk 1]
```diff
@@ -1066,12 +1057,20 @@ func (p *PluginContact) createEventSubscrption(taskID string, subTaskChan chan<-
...
+		reqJSON         string
 		err             error
 		resp            errResponse.RPC
 		percentComplete int32
 	)
 	defer wg.Done()
 
+	reqBody, err = json.Marshal(request)
+	if err != nil {
+		log.Println("error while trying to marshal create event request: ", err.Error())
```
Comment (shivacharanms) [inline svc-events/events/evtsubscription.go:1070]: Shouldn't this error be returned?

Comment (a-ajith) [reply svc-events/events/evtsubscription.go:1070]: The function doesn't have a return type. Plus as input to the Marshal function here is a well-defined struct, so there are no chances for an error to occur. Even if an error occurs, the flow won't be affected and the only difference will be JSONBody in the payload will have an empty string.

Comment (shivacharanms) [reply svc-events/events/evtsubscription.go:1070]: If an error is being captured then it should be handled.

Comment (a-ajith) [reply svc-events/events/evtsubscription.go:1070]: Error handling for the entire function has to be handled. This will be taken up as a back log item.

**Review by irappa-pattar — APPROVED** [review]
LGTM, but run the BDD to make sure it's not breaking anything.

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by hemantha-beecherla — APPROVED** [review]
LGTM

**Review by skhan-anjum — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

**Review by rameshkk-hpe — APPROVED** [review]
LGTM

**Pull request merged**